relay_block/release.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Block": "nio/relay",
```

Comment:
Does capitalization matter in these? `MergeState` was capitalized but `relay` is not